src/index.html

```diff
@@ -1,13 +1,389 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Bakerlab</title>
+    <link rel="icon" href="images/Icon-cookie.png">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=IBM+Plex+Sans:wght@300;400;500;600;700&family=Inter:wght@400;500&family=Montserrat:wght@300;400;500;700&family=Roboto&display=swap"
+      rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Hello Mate Academy</h1>
+    <header class="header">
+      <div class="container">
+        <div class="top-menu">
+          <a
+            href="#"
+            class="logo"
+            >
+            <img
+              src="images/Logo.svg"
+              alt="logo"
+              class="logo__img">
+          </a>
```

Comment:
Better change this way all over the code
```suggestion
            <img
              src="images/Logo.svg"
              alt="logo"
              class="logo__img"
            >
          </a>
```